devimint/src/tests.rs

```diff
@@ -1558,7 +1558,11 @@ pub async fn reconnect_test(dev_fed: DevFed, process_mgr: &ProcessManager) -> Re
 
     fed.start_server(process_mgr, 0).await?;
     fed.mine_then_wait_blocks_sync(100).await?;
-    fed.await_all_peers().await?;
+    poll("federation back online", None, || async {
```

Comment:
Thinking aloud: should this be just part of `await_all_peers`? Otherwise won't we hit it soon someplace else using it?

I guess if someone is using `await_all_peers` ... they can kind of expect some peers to be down, and don't want this function to fail because of it?

Reply:
That's an excellent observation 🙂